### DISCUSSION TITLE
Incorrect display of the main page of the site

### DISCUSSION THREAD

**Discussion by David2261** [community/community#55661] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I opened the main page of the site and it didn't display correctly for me. At first I thought it was because my screen was zoomed in, but I checked - it's not. This is not related to cybersecurity, as it seems to me. This is most likely a UI/UX developer problem. I would like to fix it - the screen is standard 1280px - samsung, Linux/Debian/Chrome.

![Снимок экрана от 2023-05-16 20-10-49](https://github.com/community/community/assets/65621634/e32c74bc-5ad0-4748-b349-159dd1367e68)

**Comment by lucasyvas** [comment]
Doesn't happen on Chromium for me only Firefox 🤔

**Comment by hanprat** [comment]
Mac Safari 16.4.
<img width="414" alt="Screenshot 2023-05-17 at 06 17 28" src="https://github.com/community/community/assets/810603/a67cb5e2-7c44-42fe-89e4-a2a4d919c7b6">
